.github/workflows/release.yaml

```diff
@@ -61,9 +61,13 @@ jobs:
       matrix:
         # Non-deprecated from https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
         # Note: We don't test arm64 on release as it is unlikely to fail and too much effort.
+        #
+        # Operating systems in this matrix can run recent versions of Envoy. This needs maintenance
+        # as Envoy requires newer versions of libc. The point of these tests are only to ensure the tarball wasn't corrupted
+        # or built for the wrong platform.
         include:
-          - os: ubuntu-18.04
-          - os: ubuntu-20.04
+          - os: ubuntu-20.04 # the envoy 1.24.0 is the earliest to support this.
```

Comment:
oh.. is this correct? :D 1.24 is the latest envoy version

Reply:
yeah my head is spinning. WDYT of the following. Too verbose?

```patch
diff --git a/.github/workflows/release.yaml b/.github/workflows/release.yaml
index d35718e..3c56f88 100644
--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -66,7 +66,8 @@ jobs:
         # as Envoy requires newer versions of libc. The point of these tests are only to ensure the tarball wasn't corrupted
         # or built for the wrong platform.
         include:
-          - os: ubuntu-20.04 # the envoy 1.24.0 is the earliest to support this.
+          # - os: ubuntu-18.04 # Envoy 1.23.x is the latest known Envoy version that runs on ubuntu-18.04.
+          - os: ubuntu-20.04 # is the earliest Ubuntu version that runs Envoy (since Envoy 1.24.0).
           - os: ubuntu-22.04
           - os: macos-12
           - os: windows-2019
```

Reply:
nit is the English looks better in reverse. e.g. envoy running on vs ubuntu running envoy.

Envoy 1.23.x is the last Envoy version that runs on ubuntu-18.04.
Envoy 1.24.x requires minimally ubuntu 20.04